src/backend/compiler/compiler.cc

```diff
@@ -990,38 +991,69 @@ extern "C" void seashell_preprocessor_free(struct seashell_preprocessor *preproc
 }
 
 /**
- * seashell_preprocessor_set_main_file(struct seashell_preprocessor* preprocessor, const char* file)
- * Sets the main file to begin running the preprocessor on
+ * seashell_preprocessor_set_question_dir(struct seashell_preprocessor* preprocessor, const char *dir)
```

Comment:
Do we need this for dependency resolution?  I had thought you could pass the preprocessor the same -I flags that we pass to the main clang invocation.